CHANGELOG.txt

```diff
@@ -941,7 +941,7 @@ Airflow 1.10.0, 2018-08-03
 - [AIRFLOW-1688] Support load.time_partitioning in bigquery_hook
 - [AIRFLOW-1975] Make TriggerDagRunOperator callback optional
 - [AIRFLOW-1480] Render template attributes for ExternalTaskSensor fields
-- [AIRFLOW-1958] Add **kwargs to send_email
```

Comment:
This isn't the right fix - we need to change how we pull in CHANGELOG to RST instead of needing to have a .txt file be vailid .rst

Reply:
I will cancel this change. In fact, the problem seems to be bigger and should be done with a separate PR.

The rest of the changes is correct?